01 Cloud Platform/13 Community/05 Quant League/05 Prizes.html

```diff
@@ -1 +1,2 @@
-<p>The top three organizations at the end of the quarterly competition receive prices of $500, $300, and $200, respectively. There must be at least 10 submissions in the competition to qualify for prizes.</p>
+<p>The top three organizations at the end of the quarterly competition receive prices of $500, $300, and $200, respectively. This is bound to change based on company sponsors.</p>
+<p>Additionally, the members of the winning team will get a chance to interview for a QuantConnect internship. </p>
```

Comment:
Seeking to grow prize pool with corporate sponsorship / and link to the section.